frontend/app/components/shared/LiveSessionList/LiveSessionList.tsx

```diff
@@ -41,7 +41,7 @@ function LiveSessionList(props: Props) {
   var timeoutId: any;
   const { filters } = filter;
   const hasUserFilter = filters.map((i: any) => i.key).includes(KEYS.USERID);
-  const sortOptions = [{ label: 'Freshness', value: 'timestamp' }].concat(
+  const sortOptions = [{ label: 'Start Time', value: 'timestamp' }].concat(
```

Comment:
Unrelated to spots

Reply:
Should I create a separate branch for this? What is the right approach?

Reply:
yep. This branch is for spots ui